unit1/lesson12.rst

```diff
@@ -0,0 +1,41 @@
+Unit 1 - Lesson 12 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Create static HTML pages using a variety of different tags inside the body.
+- Create an account on Github and create hello world repository.
+- Understand what Git is and why programmers use it.
+
+Objectives
+----------
+
+HTML
+^^^^
+
+- Know what HTML stands for
+- Describe the components of HTML tag structure
+- Describe what an HTML element is, and how an element is related to a tag
+- Name, describe the purpose of, and use common HTML tags: <b>, <em>, <a>, <img>, <br>, <p>, <span>, <div>, <form>
+- Define HTML attributes and how they are used
+- Describe how whitespace in HTML documents effects rendering
+- Describe inline and block display; for common elements identify their default display
```

Comment:
This feels like it would be better placed in the CSS section.

Reply:
Ok! What about moving whitespace and inline and block display to CSS? I think tags and attributes go naturally together and it is easier to learn them as part of HTML.

Reply:
My comment was only in reference to the inline vs. block stuff. The rest seems to make sense when covering HTML.